app/helpers/quadicon_helper.rb

```diff
@@ -216,6 +221,48 @@ def render_quadicon_label(item, row)
     end
   end
 
+  # Renders a quadicon for PhysicalServer
+  #
+  def render_physical_server_quadicon(item, options)
+    output = []
+    if settings(:quadicons, :physical_server)
+      output << flobj_img_simple("layout/base.png")
+
+      output << flobj_p_simple("a72", (item.host ? 1 : 0))
+      output << flobj_img_simple("svg/currentstate-#{h(item.power_state.downcase)}.svg", "b72")
```

Comment:
However, I read about a concern that @Fryguy had. https://github.com/ManageIQ/manageiq/pull/14287

@juliancheal sent this PR and was accepted but I don't have sure if this PR override power_state or not. According with this PR he jut added a migration with a new property called `raw_power_state`